code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m

```diff
@@ -29,3 +29,7 @@
     xlabel('Hemisphere Injected');  % weird, right??
     ylabel(ax, 'Sum squared error');
     title(ax, sprintf('%s\n%s\n ', taskTitle, stimSet));
+    
+    yrule = ax.YAxis;
+    yrule.FontSize=15
```

Comment:
mmmm I doubt this is compatible with my version of MATLAB, and inconsistent with what we do elsewhere. I'd rather see `ylabel('blah', 'fontsize', 15);` or `set(gca, 'FontSize', 15);`